integration/build/build_squash_test.go

```diff
@@ -86,8 +88,11 @@ func TestBuildSquashParent(t *testing.T) {
 		container.WithImage(name),
 		container.WithCmd("/bin/sh", "-c", "cat /hello"),
 	)
+
+	poll.WaitOn(t, container.IsStopped(ctx, client, cid), poll.WithDelay(100*time.Millisecond))
 	reader, err := client.ContainerLogs(ctx, cid, containertypes.LogsOptions{
 		ShowStdout: true,
+		ShowStderr: true,
```

Comment:
Same as https://github.com/moby/moby/pull/48716/commits/cb4b8168568a6abbc5c90c2ecbac4db6b9bf8aaa#r1809141990, I guess, i.e. this is not needed.